numba_cuda/numba/cuda/_internal/cuda_bf16.py

```diff
@@ -23,8 +23,8 @@
     make_attribute_wrapper,
     register_model,
 )
-from numba.core.typing import signature
-from numba.core.typing.templates import AttributeTemplate, ConcreteTemplate
+from numba.cuda.typing import signature
```

Comment:
This is an autogenerated file, so we need to update Numbast to reflect these changes. cc @isVoid